### PR TITLE
Switched to the 2.0 version of the apt module

### PR DESCRIPTION
**Pull request by tedivm**
This updates the usages of the apt-get module.

### CODE REVIEW

**Comment by mullr** [comment]
cc @mullr

**Comment by mullr** [comment]
@tedivm The PR triage team is meeting tomorrow morning, and this is on their list.

**Comment by tedivm** [comment]
Awesome :-)

You should get them to check out #626 too so this module can be up to date with the new concat one as well.

File: manifests/repo/apt_postgresql_org.pp [hunk 1]
```diff
@@ -10,12 +10,17 @@
...
-    location    => 'http://apt.postgresql.org/pub/repos/apt/',
-    release     => "${::lsbdistcodename}-pgdg",
-    repos       => "main ${postgresql::repo::version}",
-    key         => 'B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8',
-    key_source  => 'https://www.postgresql.org/media/keys/ACCC4CF8.asc',
-    include_src => false,
+    location => 'http://apt.postgresql.org/pub/repos/apt/',
+    release  => "${::lsbdistcodename}-pgdg",
+    repos    => "main ${postgresql::repo::version}",
+    key      => {
```
Comment (cyberious) [inline manifests/repo/apt_postgresql_org.pp]: This is not backwards compatible with apt 1.8, needs to bump metadata to use apt 2.x

**Comment by mhaskel** [comment]
@tedivm would you be ok with this waiting a few weeks for the new version of apt? Then the change can just be simplified to a metadata change :)

**Comment by tedivm** [comment]
The downside to that is I won't be able to use the puppetdb module, as it depends on this postgres module and I can't install this postgres module without causing dependency conflicts. Waiting several weeks for puppetdb means pushing my whole deployment back.

**Comment by tedivm** [comment]
Frankly, I think you guys should skip this- and everything else you're doing- to try to slip out an apt module sooner than a few weeks agrom now that doesn't break everyone's projects.

**Comment by mhaskel** [comment]
@tedivm thanks, but I'm going to close this PR for now. We're working on the apt 2.1.0 release which will allow this module to use 1.x or 2.x, and it will only require updating the metadata binding. Once apt is released we'll  add a new PR to fix that.

**Pull request closed without merging**